### DISCUSSION TITLE
[Recruit Window] Default behavior of Recruit Window

### DISCUSSION THREAD

**Discussion by TheRealIndru** [ihhub/fheroes2#8249] [Q&A]
Hi,

Currently, the default behavior of the recruiting window seems to be to pre-click the MAX button and have already the creature count set to the maximum possible the player can recruit.

![image](https://github.com/ihhub/fheroes2/assets/6568040/0d723f22-557c-4a64-9cea-dbd7b2b7a7b4)

Is this feature configurable somewhere?

As a player coming from vanilla Heroes 2, this is annoying to me and kind of a deal breaker, as I always attempt to click the MAX button, as my reflex was built on years and years of vanilla HOMAM 2, and thus end up recruiting only one creature (as it switches to the minimum since it's now the MIN button)

I would like to deactivate this as it's a pain to go back and forth and correct the mistakes.

**Comment by ihhub** [comment]
Hi @TheRealIndru , forgetting for a moment your past experience with the original game, can you tell me which way is better and more convenient for players?

**Reply by TheRealIndru** [reply]
Hi @ihhub ,
I obviously can't speak for other players, but for me the convenient way is the vanilla behavior. So I would like to be able to toggle it.
Thank you!

**Reply by ihhub** [reply]
Hi @TheRealIndru , I am not asking you how you are used to play. I am asking your opinion on how players see such UX design. Hence, I emphasized **forgetting for a moment your past experience with the original game**.

**Reply by TheRealIndru** [reply]
There are probably players who like it, and players who do not.

**Comment by Branikolog** [comment]
Hi, @TheRealIndru !
I've heard a similar opinion from old and experienced players, but they completely changed their mind later. Lots of people are even happy with this behaviour (I've watched a different of streams and spoke to hundreds of players from our community).
Just give it a try - you can get used to it in just a few days. Buying a maximum number of troops is more convenient and fast way to play. In 95% of cases a player trends to buy maximum he can. Adding an option for 2-3 persons based just on their habits is not really worth the time of the development team spent on it. There's a plenty of more important and awaited things, like the editor or AI, that should be prioritised at the current moment.

**Reply by TheRealIndru** [reply]
Agree to disagree... Doesn't harm anyone if it's configurable... I can only point it out, yes. It's up to them to add it or not.

**Reply by oleg-derevenetz** [reply]
> Doesn't harm anyone if it's configurable...

If you make every little thing of such kind "configurable", then the result will be a bunch of screens with hundreds of settings that will be used by two and a half people, but it will require efforts for development and further support. Why do you think it wasn't made "configurable" in the original game?
